internal/service/cloudbackupschedule/data_source_cloud_backup_schedule.go

```diff
@@ -247,67 +261,46 @@ func DataSource() *schema.Resource {
 
 func dataSourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	connV220231115 := meta.(*config.MongoDBClient).AtlasV220231115
+	connV2 := meta.(*config.MongoDBClient).AtlasV2
 
 	projectID := d.Get("project_id").(string)
 	clusterName := d.Get("cluster_name").(string)
+	useZoneIDForCopySettings := false
```

Comment:
knit: you can also declare like the next vars: 
var useZoneIDForCopySettings bool

or even create a var ( ... ) group to define all vars here